cmd/secrets-provider/main.go

```diff
@@ -34,8 +34,14 @@ func main() {
 		printErrorAndExit(messages.CSPFK015E)
 	}
 
+	// verify CONTAINER_MODE and SUPPORT_MULTIPLE_APPS are not both set
+	envValMultiApps := os.Getenv("SUPPORT_MULTIPLE_APPS")
+	if authnConfig.ContainerMode != "" && (envValMultiApps != "false" && envValMultiApps != "") {
```

Comment:
why can't both live together?

Reply:
updated to your idea but to answer you here. container_mode init and support multiple apps = true cannot run together because they are different flows. One is for init, the other a Job